### DISCUSSION TITLE
Neutral Lut DDS?

### DISCUSSION THREAD

**Discussion by Trainmaster9977** [ScanMountGoat/Smush-Material-Research#43] [Q&A]
Hello- I would ask about this on the Smush Lut repo, but that doesn't have a discussions page unless I somehow missed it.

Been busy this weekend, but finally getting back around to testing out rendering stuff like you recommended. However, I'm running into a slight problem- being unsure how much the current LUT is affecting things. Say, me checking a color that was inputted as 0-20-20, and coming out as 0-28-28. 

While I'll be fine with the cell shader, or any other custom shaders, being affected by the LUT, I also do want a good neutral environment for testing.

Now, I would normally go to your smush-lut repo and just use the neutral LUT on there. However, if I remember one of our previous conversations correctly, and read the documentations on the repo correctly, the program does adjust colors a bit once you input something. And given I want an accurate baseline, I'm a bit worried about that. 

Which then leads to me wanting to use ultimate-tex as an alternative way to insert it. But well...given your neutral lut png and the output dds are in different formats entirely, with different dimensions and layouts, that's only so far that will get me.

So I want to ask if you have a dds version of your neutral lut.

By the way, apologies if there is already a perfectly neutral lut that exists. I compared your neutral lut to a random lut out of the folder I converted a while back, and it was close but not identical. So just decided that it would be better to ask ya about this instead of seeing if there is another lut out of the couple hundred that are neutral ones. 

Thanks!

**Comment by ScanMountGoat** [comment]
If you only need to use RenderDoc, just edit out the code that uses the LUT. You can also just use the provided identity LUT and use the --raw option with smush_lut. The color correction is necessary if you want to use a non identity LUT. It's not possible to get perfect accuracy in this case, and the program will do a better job accounting for the post processing code in game than you can do by hand.

**Reply by Trainmaster9977** [reply]
Renderdoc is a bit difficult to use for this because I'm trying to test out different texture adjustments and the like. Quick and changing textures are not things renderdoc knows about lol. 

Also, by identity lut, you mean the neutral lut, right?

I remember having a conversation with you months ago about how a LUT of mine was changed a bit even with --raw. Wouldn't that still apply to the identity lut being passed through it?

I just want a baseline that I know is fully neutral.

**Reply by ScanMountGoat** [reply]
Raw won't apply any color edits. You can replace texture acceses in RenderDoc with a hardcoded value. It's much faster than changing textures and launching the game. For an identity LUT, just replace the texture access in the shader with the input RGB value.
